app/not-found.jsx

```diff
@@ -0,0 +1,28 @@
+import { ogMeta, twitterMeta } from '@data/metadata'
+
+
+export async function generateMetadata() {
+  const description = `Uh-oh! It appears that this page doesn't exist!`
+  return {
+    title: `404 Page Doesn't Exist | HyperUI`,
+    description: description,
+    openGraph: {
+      title: `404 Page Doesn't Exist | HyperUI`,
+      description: description,
+      ...ogMeta,
+    },
+    twitter: {
+      title: `404 Page Doesn't Exist | HyperUI`,
+      description: description,
+      ...twitterMeta,
+    },
+  }
+}
+
+export default function Custom404() {
+  return (
+    <div className="grid h-screen px-4 bg-white place-content-center">
+      <h1 className="tracking-widest text-gray-500 text-center uppercase text-3xl">404 | URL Not Found!</h1>
```

Comment:
This isn't how the component I linked looks.

Reply:
> https://www.hyperui.dev/components/application-ui/error-pages#component-1

You linked to components#1 (simple), I just added  ` text-center `, now it's the same to same component you linked, sorry